web-bluetooth/index.html

```diff
@@ -33,6 +34,7 @@ <h3>Combining multiple operations</h3>
 
 <h3>Scanning</h3>
 <p><a href="scan.html">Scan for Advertisements</a> - Basic scan for advertisements.</p>
+<p><a href="watch-advertisements.html">Watch Advertisements (Promises)</a> / <a href="watch-advertisements-async-await.html">Watch Advertisements (Async Await)</a> - watch advertisements and connect to permitted Bluetooth devices.</p>
```

Comment:
This is the only watchAdvertisements() change I see. Perhaps you forgot to include this file in the commit?

Reply:
I did. It is now fixed. Thanks!